lib/constants.js

```diff
@@ -1,3 +1,5 @@
 export const ONE_MINUTE_IN_MILLISECONDS = 60_000
 export const MEASUREMENT_DELAY = ONE_MINUTE_IN_MILLISECONDS
 export const RETRIEVE_TIMEOUT = 10_000
+export const SUI_NETWORK = 'testnet'
+export const WALRUS_OBJECT_ID = '0x161434e1e4bf91f0d55111602d407039394bb9a72d3e90e74d1ee26f2b16c7b5'
```

Comment:
This variable isn't self explanatory to me, could you please add a comment?

Reply:
Great point, it was a bit vague. I've changed the constant name and added a comment

https://github.com/CheckerNetwork/walrus-checker/pull/15/commits/4865e8c25bec5844aa98b55d145730bca53925b0